rc/extra/git-tools.kak

```diff
@@ -146,6 +146,79 @@ Available commands:\n  add\n  rm\n  blame\n  commit\n  checkout\n  diff\n  hide-
         '
     }
 
+    jump_next_hunk() {
```

Comment:
Thats around 80 new lines for next/prev, I would be more confortable with less, I think there should be a bit of room for simplification, and hopefully sharing code between the two implementation.